### PR TITLE
adding a new entry: Server Support Video

### PR DESCRIPTION
**Pull request by rusliabdulgani**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by rusliabdulgani** [comment]
Hi, 
Please review of my pull request..

Thank you

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@rusliabdulgani Can you rewrite this to avoid first- and second-person pronouns ("We" and "you"). We try to avoid these in docs. Once you do this tag me and I'll review this again. Also you need to sign the [CLA](https://cla-assistant.io/Codecademy/docs?pullRequest=3309).

**Comment by rusliabdulgani** [comment]
Hi @SSwiniarski , can you re-review of my pull request please?

File: content/html/concepts/videos/videos.md
Comment (SSwiniarski) [inline content/html/concepts/videos/videos.md:37]:
```suggestion
## Server Support of Video

It is necessary to add configuration in the web server to enable video to show in a browser. What is needed is to add the video MIME type to the server configuration. 
```

File: content/html/concepts/videos/videos.md
Comment (SSwiniarski) [inline content/html/concepts/videos/videos.md:39]:
```suggestion
If the web server is Apache and it's serving OGG video, it's enabled by adding the "video/ogg" MIME type. Edit the "mime.types" file in "/etc/apache" or use the "AddType" configuration directive in httpd.conf
```

File: content/html/concepts/videos/videos.md
Comment (SSwiniarski) [inline content/html/concepts/videos/videos.md:47]:
```suggestion
If the website need serve video as WebM, it can be done by adding "video/webm" to the "mime.types" in "/ect/apache" or add via the "AddType" configuration directive in httpd.conf
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@rusliabdulgani a few more comments.

File: content/html/concepts/videos/videos.md
Comment (rusliabdulgani) [reply content/html/concepts/videos/videos.md:47]: @SSwiniarski already fix them, please review.. thanks

**Review by SSwiniarski — APPROVED** [review]
@rusliabdulgani I committed the suggestion you missed. Now it looks good for a second review.

**Comment by rusliabdulgani** [comment]
> @rusliabdulgani I committed the suggestion you missed. Now it looks good for a second review.

**Comment by rusliabdulgani** [comment]
@SSwiniarski means i need wait for another reviewer, isn't it? sorry if i ask this, cause this is my first time to contribute

**Comment by SSwiniarski** [comment]
Yes, all PRs go through two reviews before merging. Did you intend to close this PR?

**Comment by rusliabdulgani** [comment]
and how can i request another reviewer to review this PR ? i think so far just you that already review my pull request.

**Comment by SSwiniarski** [comment]
@rusliabdulgani you closed this PR, so there's going to be no further review of it. Did you close it by mistake?

**Comment by KTom101** [comment]
@rusliabdulgani I'm going to open this PR back up. Please note, however, that you did not do what the issue was asking for. You were supposed to add images to show what the examples would look like when executed in HTML. The section you added is not relevant to the issue.

**Review by yangc95 — CHANGES_REQUESTED** [review]
@rusliabdulgani can you add an image of the last example code block lines (68-75). Please place the new image below the block; as well an image for the code block on lines 112-132?

Thank you 🙂

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
Apologies, this does not fit in with the entry. However, I would recommend looking for other available issues that are relevant. Thank you!

**Pull request closed without merging**
